DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/AdminUserDataServiceTests.cs

```diff
@@ -13,7 +14,7 @@ public partial class UserDataServiceTests
         public void GetAdminUserById_Returns_admin_user()
         {
             // Given
-            var expectedAdminUser = UserTestHelper.GetDefaultAdminUser();
+            var expectedAdminUser = UserTestHelper.GetDefaultAdminUser(active: true);

```

Comment:
`GetDefaultAdminUser()` gets the user which has been pre-added to the test database, but only as long as we don't override properties on it. So it's better not to have `active: true`. Same in test below.

Reply:
Earlier Active field wasn't being set, so it was always returns false, and all assertions used it and expect it to be false. When we set this field now, it will return true for some records and the test will fail if this returns False by default before. However, when I ran tests on latest data without this arugument, all tests pass ok, so removed it as we may don't need it. However, we should be careful when using assertion on active field since it will now return either True or false (not just false as it did before)